mkt/settings.py

```diff
@@ -381,7 +381,7 @@
 # The verification URL, the addon id will be appended to this. This will
 # have to be altered to the right domain for each server, eg:
 # https://receiptcheck.addons.mozilla.org/verify/
-WEBAPPS_RECEIPT_URL = '/verify/'
+WEBAPPS_RECEIPT_URL = 'http://mp.dev/receipt-verifier/'
```

Comment:
Can't you use [SITE_URL](https://github.com/andymckay/zamboni/blob/1109334/mkt/settings.py#L367) since it is overridable by env var and should default to ``mp.dev``?